torch/functional.py

```diff
@@ -433,20 +441,28 @@ def unique(input, sorted=True, return_inverse=False, dim=None):
 
     """
     if dim is not None:
-        output, inverse_indices = torch._unique_dim(
+        output, inverse_indices, counts = torch._C._VariableFunctions.unique_dim(
```

Comment:
maybe torch.unique_dim ?

Reply:
@VitalyFedyunin I don't think `torch.unique_dim`, because we don't want to expose `unique_dim` to Python, because we already have `unique(dim=...)`. See:
https://github.com/pytorch/pytorch/pull/18649/files#diff-61b893a141a037c275f12eba9a9b864fR388